cluster/main.tf

```diff
@@ -12,11 +12,13 @@ locals {
 }
 
 module "ecs_cluster" {
+  #checkov:skip=CKV_AWS_79:Cloudposse module allows configuration that ensures passing this check
```

Comment:
Perhaps you could update the message that this is ensured by setting `metadata_http_tokens_required   = true`

Reply:
the checkov comment is still not updated; it's a useful info to know up front, rather than having have to dig into the code to understand why the checkov is not happy here

Reply:
I think the snipped above is being shown from the point at which it was originally reviewed:
Updated comment here:
https://github.com/ministryofjustice/modernisation-platform-terraform-ecs-cluster/blob/ccb2f87b1bdafd32a9b4053315463ff7960c227e/cluster/main.tf#L2

and explicit enabling of IMDSv2 here:
https://github.com/ministryofjustice/modernisation-platform-terraform-ecs-cluster/blob/ccb2f87b1bdafd32a9b4053315463ff7960c227e/cluster/locals.tf#L10-L11

and here

https://github.com/ministryofjustice/modernisation-platform-terraform-ecs-cluster/blob/ccb2f87b1bdafd32a9b4053315463ff7960c227e/cluster/variables.tf#L60-L71

Reply:
cool, thanks George!